packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -29,6 +29,8 @@ type ALUIEvent<T = EventHandlerMap> = {
     element: HTMLElement | null,
     // Element text extracted from element
     elementName?: string | null,
+    // The source attribute where we got the elementName from
+    elementNameSource?: string | null,
```

Comment:
I thought we want this data to be part of the metadata, but if you want to add it as a explicit field, better be more precise
`elementNameSource: ALElementNameResult['source'] | null;`